Integrations/integration-QRadar.yml

```diff
@@ -59,7 +64,8 @@ script:
     var server = params.server+'/';
     var insecure = params.insecure;
     var query = params.query;
-    var incidentsPerCall = 50;
+    var offensesPerCall = parseInt(params.offensesPerCall);
```

Comment:
what's the risk when you change a hardcoded var to a parameter?

Reply:
@omercnet - I think this can be a risk of server performance if we ingest large number of incidents. when it was limited to 50 - what was the experience - we got only the newer 50, and the older were discarded? Can we do this so we will ingest limited number (starting with the older ones) and in the next iteration continue on the rest?

Reply:
also - need to make sure this doesn't break existing setup. i.e. if a customer already have this integration set up - this parameter is empty. what happens then?

Reply:
ah. just saw default... ignore :)

Reply:
Performance concern is valid but QRadar offenses API results are sorted descending by date and there's no way to override it

If someone disables the integration for some time and enables it with lots of new offense they will all be created as incidents immediately